R/guide-axis.R

```diff
@@ -509,41 +518,42 @@ axis_label_priority_between <- function(x, y) {
 #' @noRd
 #'
 axis_label_element_overrides <- function(axis_position, angle = NULL) {
-  if (is.null(angle)) {
+
+  if (is.null(angle) || is.waive(angle)) {
     return(element_text(angle = NULL, hjust = NULL, vjust = NULL))
   }
 
-  # it is not worth the effort to align upside-down labels properly
-  check_number_decimal(angle, min = -90, max = 90)
```

Comment:
I very much did need some alignments for upside-down labels, so I have expanded this function to include this.